doc/api/url.md

```diff
@@ -1262,6 +1262,12 @@ A `TypeError` is thrown if `urlString` is not a string.
 
 A `URIError` is thrown if the `auth` property is present but cannot be decoded.
 
+Use of the legacy `url.parse()` method is not recommended. All users should
+migrate to the WHATWG `URL` api. Because the `url.parse()` method uses a
+lenient, non-standards compliant algorithm for parsing URL strings, security
```

Comment:
```suggestion
lenient, non-standard algorithm for parsing URL strings, security
```
Take-it-or-leave-it suggestion but I expect this'll be a little easier to read for non-native speakers.